package.json

```diff
@@ -177,7 +177,24 @@
             "description": "Nexus IQ password (default: $IQ_TOKEN from environment)"
           }
         }
+      },
+      {
+        "type": "object",
+        "title": "Sonatype OSSIndex configuration",
+        "properties": {
+          "ossindex.username": {
+            "type": "string",
+            "default": "admin",
```

Comment:
Not sure, but I would expect the default OSSI credentials to be empty (where only 'admin'/'admin123' make sense for IQ).

Reply:
Correct! Should be empty by default!